frontend/widgets/OBSBasic.hpp

```diff
@@ -63,6 +63,7 @@ class YouTubeAppDock;
 class QMessageBox;
 class QWidgetAction;
 struct QuickTransition;
+class ScreenshotObj;
```

Comment:
Place this between `OBSProjector` and `VolControl`.

Reply:
Fixed